.github/workflows/main.yml

```diff
@@ -6,172 +6,200 @@ name: DISC
+#        GITHUB_PULL_REQUEST: ${{ github.event.number }}
+#        LOCAL_DEV_DOCKER: disc-dev-cuda11.0
+#        REMOTE_DEV_DOCKER: bladedisc:latest-devel-cuda11.0
+#        REMOTE_RUNTIME_DOCKER: bladedisc:latest-runtime-tensorflow2.4
+#        RUNTIME_BASEIMAGE: tensorflow/tensorflow:2.4.0-gpu
+#      run: |
+#        set -e
+#        bash ./scripts/ci/deploy_tf_wrapper.sh
+#  CUDA11-TORCH171:
+#    if: github.repository == 'alibaba/BladeDISC'
+#    # The type of runner that the job will run on
+#    runs-on: [self-hosted, gpu-t4]
+#    # Steps represent a sequence of tasks that will be executed as part of the job
+#    steps:
+#    # Runs a single command using the runners shell
+#    - name: Checkout
+#      uses: actions/checkout@v2.4.0
+#    - name: Build Dev Docker
+#      run: |
+#        set -e
+#        git submodule sync
+#        git submodule update --depth=1 --init --recursive
+#        docker build -t disc-dev-cuda11.0 \
+#          --build-arg BASEIMAGE=nvidia/cuda:11.0-cudnn8-devel-ubuntu18.04 -f docker/dev/Dockerfile .
+#    - name: Build and Test DISC
+#      run: |
+#        set -e
+#        nvidia-docker run --rm -t --user $(id -u) \
+#          -v $HOME/.cache:$HOME/.cache \
+#          -v /etc/passwd:/etc/passwd:ro \
+#          -v /etc/group:/etc/group:ro \
+#          -v $PWD:/disc \
+#          -e GITHUB_WORKFLOW=$GITHUB_WORKFLOW \
+#          -w /disc \
+#          disc-dev-cuda11.0 bash ./scripts/ci/test_pytorch_blade.sh
+#    - name: Deploy PyTorch Blade
+#      if: github.event.ref == 'refs/heads/main'
+#      env:
+#        ALIYUN_DOCKER_USERNAME: ${{ secrets.ALIYUN_DOCKER_USERNAME }}
+#        ALIYUN_DOCKER_PASSWORD: ${{ secrets.ALIYUN_DOCKER_PASSWORD }}
+#        DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+#        DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+#        GITHUB_PULL_REQUEST: ${{ github.event.number }}
+#        LOCAL_DEV_DOCKER: disc-dev-cuda11.0
+#        REMOTE_DEV_DOCKER: bladedisc:latest-devel-cuda11.0
+#        REMOTE_RUNTIME_DOCKER: bladedisc:latest-runtime-torch1.7.1
+#        RUNTIME_BASEIMAGE: nvidia/cuda:11.0-cudnn8-devel-ubuntu18.04
+#      run: |
+#        set -e
+#        bash ./scripts/ci/deploy_pytorch_blade.sh
+#  CPU-TF115:
+#    uses: ./.github/workflows/cpu_reusable.yml
+#    with:
+#      remote_runtime_docker: bladedisc:latest-runtime-tensorflow1.15-cpu
+#      runtime_base_image: tensorflow/tensorflow:1.15.5
+#      exec_command: bash ./scripts/ci/build_and_test.sh --cpu-only
+#      deploy_command: bash ./scripts/ci/deploy_tf_wrapper.sh
+#    secrets:
+#      ALIYUN_DOCKER_USERNAME: ${{ secrets.ALIYUN_DOCKER_USERNAME }}
+#      ALIYUN_DOCKER_PASSWORD: ${{ secrets.ALIYUN_DOCKER_PASSWORD }}
+#      DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+#      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+#  CPU-TORCH181:
+#    uses: ./.github/workflows/cpu_reusable.yml
+#    with:
+#      extra_envs: -e TORCH_BLADE_BUILD_WITH_CUDA_SUPPORT=OFF
```

Comment:
Nit: Delete these codes if we don't need them

Reply:
We should remove the comment symbol when merging to main in the future, so I keep these jobs in the current feature branch.